Zooniverse/Snakefile

```diff
@@ -31,8 +37,42 @@ rule predict_birds:
 rule combine_predicted_birds:
     input:
         expand("/blue/ewhite/everglades/predictions/{year}/{site}/{flight}_projected.shp",
-               zip, site=ORTHOMOSAICS.site, year = ORTHOMOSAICS.year, flight=ORTHOMOSAICS.flight)
+               zip, site=SITES, year=YEARS, flight=FLIGHTS)
     output:
         "/blue/ewhite/everglades/EvergladesTools/App/Zooniverse/data/PredictedBirds.zip"
     shell:
         "python combine_bird_predictions.py"
+
+def flights_in_year_site(wildcards):
+    basepath = "/blue/ewhite/everglades/predictions"
+    flights_in_year_site = []
+    for site, year, flight in zip(SITES, YEARS, FLIGHTS):
+        if site == wildcards.site and year == wildcards.year:
+            flight_path = os.path.join(basepath, year, site, f"{flight}_projected.shp")
+            flights_in_year_site.append(flight_path)
+    return flights_in_year_site
+
+rule detect_nests:
+    input:
+        flights_in_year_site
+    output:
+        "/blue/ewhite/everglades/detected_nests/{year}/{site}/{site}_{year}_detected_nests.shp"
```

Comment:
Can we move the paths to the top of the file? 
eg
DETECTED_NEST = "/blue/ewhite/everglades/detected_nests/{year}/{site}/{site}_{year}_detected_nests.shp"

Reply:
No, that will break the wildcard system. I'm free to Zoom for a bit and look at what you're seeing.